### PR TITLE
Updated Dismount-DiskImage to handle PassThru

### PR DESCRIPTION
**Pull request by AdamGrossTX**
Dismount-DiskImage doesn't support -PassThru in newer PowerShell versions. I added logic to test whether the -PassThru parameter is supported and if not, remove the -PassThru parameter from the command.

### CODE REVIEW

File: hyperv-tools/Convert-WindowsImage/Convert-WindowsImage.psm1 [hunk 1]
```diff
@@ -1584,7 +1584,7 @@ You can use the fields below to configure the VHD or VHDX that you want to creat
                     $VhdPath      = Join-Path  -Path $VhdPath -ChildPath $vhdNameTemp
                 }
 
-                Write-Debug -Message "  Temporary $VhdFormat path is: `"$VhdPath`""
+                Write-Debug -Message "  Temporary $VhdFormat path is: `�$VhdPath`�"
```
Comment (tabs-not-spaces) [inline hyperv-tools/Convert-WindowsImage/Convert-WindowsImage.psm1]: Heaps of corrupts on double quotes - will need to fix these before the PR would go through.

Comment (AdamGrossTX) [reply hyperv-tools/Convert-WindowsImage/Convert-WindowsImage.psm1]: Thanks for the find. Must've pasted in RTF or something on the PR. I've updated with a new commit.

**Comment by AdamGrossTX** [comment]
Can I get an update on the status of this PR?

**Comment by aricbernard** [comment]
Why is this issue still not resolved?  The latest module published to PSGallery is 10.0 released in 2016.  Your public documentation for Azure Stack HCI SDN uses this cmdlet, but the command fails on Windows 11.

**Pull request merged**